.changeset/proud-schools-reply.md

```diff
@@ -0,0 +1,42 @@
+---
+`@spectrum-css/swatchgroup`: major
+`@spectrum-css/swatch`: major
+---
+
+#### S2 migration swatchgroup
+
+This migrates the `swatch` and `swatchgroup` components to S2. Custom properties have been remapped and added per the design specification.
+
+An `Add Swatch` variant has been added with the required color tokens and the specified add UI icon.
+
+##### Removed custom properties
+
+`--spectrum-swatch-border-color`
+`--spectrum-swatch-dash-icon-color`
+
+##### New custom properties
+
+`--spectrum-swatch-border-color-rgb`
+`--spectrum-swatch-border-opacity`
+`--spectrum-corner-radius-full`
+`--spectrum-corner-radius-none`
+`--spectrum-swatch-disabled-icon-border-opacity`
+`--spectrum-swatch-icon-color`
+`--spectrum-swatch-rectangle-width-multiplier`
+`--spectrum-swatchgroup-border-color`
+
+##### New mods
+
+`--mod-add-button-background`
+`--mod-add-button-background-down`
+`--mod-add-button-background-hover`
+`--mod-add-button-background-keyboard-focus`
+`--mod-corner-radius-full`
+`--mod-mixed-button-background`
+`--mod-swatchgroup-border-color`
+`--mod-swatch-border-color-rgb`
+`--mod-swatch-border-opacity`
+
+##### Removed mods
+
+`--mod-swatchgroup-spacing-compact`
```

Comment:
Can we include a "instead, please use..." part here for those using this mod in their systems currently?